test/jdk/tools/jpackage/share/ErrorTest.java

```diff
@@ -88,6 +92,29 @@ enum Token {
 
             return appImageCmd.outputBundle().toString();
         }),
+        INVALID_MAC_JDK_BUNDLE(cmd -> {
+            // Missing "Contents/MacOS/libjli.dylib"
+            try {
+                final Path runtimePath = TKit.createTempDirectory("invalidJDKBundle");
```

Comment:
The name "invalid-jdk-bundle" fits better in dash-case naming scheme for path names in jpackage tests.